### PR TITLE
AndroidX 202002

### PR DESCRIPTION
**Pull request by moljac**
### Google Play Services Version (eg: 8.4.0):

GPS-FB bindings from 202002 -  AndroidX dependencies

### CODE REVIEW

File: source/com.google.android.gms/play-services-games/Additions/IPendingResultExtensions.cs [hunk 1]
```diff
@@ -225,96 +225,6 @@ public static async Task<ITurnBasedMultiplayerUpdateMatchResult> TakeTurnAsync (
...
-    }
-}
-namespace Android.Gms.Games.Request
-{
-    public static partial class IRequestsExtensions
-    {
-        public static async Task<IRequestsUpdateRequestsResult> AcceptRequestAsync (this IRequests api, GoogleApiClient apiClient, string requestId)
-        {
-            return (await api.AcceptRequest (apiClient, requestId)).JavaCast<IRequestsUpdateRequestsResult> ();
-        }
```
Comment (mattleibow) [inline source/com.google.android.gms/play-services-games/Additions/IPendingResultExtensions.cs:257]: I see this being removed, but the diff still has it? Is there some new automatic way of doing something?

**Review by mattleibow — COMMENTED** [review]
Everything looks good. Probably need another set of eyes on this huge diff....

The only Q I have is the removed Async methods.

File: source/com.google.android.gms/play-services-games/Additions/IPendingResultExtensions.cs (hunk 1 above)
Comment (moljac) [reply source/com.google.android.gms/play-services-games/Additions/IPendingResultExtensions.cs:257]: > Is there some new automatic way of doing something?

No. for sure.

I checked `IRequestsUpdateRequestsResult ` this and few other interfaces were removed. Not sure what is going on with diff. Could it be comparing with latest version (prerelease on nuget.org)

Comment (moljac) [reply source/com.google.android.gms/play-services-games/Additions/IPendingResultExtensions.cs:257]: Let us merge those few releases

**Pull request merged**